cmd/algod/main.go

```diff
@@ -297,6 +297,7 @@ func run() int {
 	}
 
 	err = s.Initialize(cfg, phonebookAddresses, string(genesisText))
+	algod.IsDBSchemeFinished = true
```

Comment:
Should this be a "phase" variable indicating the phase of startup rather than a series of boolean checkpoint variables?